parsers/IN_UP.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+import re
+from ast import literal_eval
+from operator import itemgetter
+import json
+from arrow import get
+from requests import Session
+import arrow
+from parsers import countrycode
+from parsers.lib import web
+
+
+def fetch_data(country_code, session=None):
+    countrycode.assert_country_code(country_code, 'IN-UP')
```

Comment:
Since this parser will only get data for Uttar Pradesh you can set countrycode='IN-UP' in the function and get rid of this line.